test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -270,6 +270,81 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | should be ([char]0x1b)
     }
 
+    Context "Test Unicode escape sequences." {
+        # These tests require the file to be saved with a BOM.  Unfortunately when this UTF8 file is read by
+        # PowerShell without a BOM, the file is incorrectly interpreted as ASCII.
+        It 'Test that Unicode escape sequence `u2195 in string returns the ↕ character.' {
+            $result = ExecuteCommand '"foo`u2195abc"'
+            $result | should be "foo↕abc"
+        }
+
+        It 'Test that Unicode escape sequence `u2195 in here string returns the ↕ character.' {
+            $result = ExecuteCommand ("@`"`n`n" + 'foo`u2195abc' + "`n`n`"@")

```

Comment:
`` `n`` is newline and I already tried using [environment]::newline.  Didn't work.  @lzybkr informed me that here string parsing doesn't rely on OS specific newline but what has been found to be the newline used in previous here strings in the file.  

BTW there is already another test for single quoted here strings below this one:
```
        It 'Test that Unicode escape sequence in single quoted is not processed.' {
            $result = ExecuteCommand '''foo`u2195abc'''
            $result | should be 'foo`u2195abc'
        }
```